src/altinn-app-frontend/webpack.config.development.js

```diff
@@ -14,10 +16,16 @@ module.exports = {
       ...common.module.rules,
       {
         test: /\.tsx?/,
+        exclude: /node_modules/,
```

Comment:
I thought this was excluded by default, but that may have been an incorrect assumption? If it is not, then this should probably be added to the production config as well?